rmw_zenoh_cpp/src/detail/zenoh_router_check.hpp

```diff
@@ -1,30 +0,0 @@
-// Copyright 2023 Open Source Robotics Foundation, Inc.
```

Comment:
imo this PR shouldn't make such big changes in this PR. Are we not able to simply port this function to zenoh-cpp?

Reply:
This is not used anymore, there is something on zenoh_cpp that replace this